airbyte-integrations/connectors/source-clockify/source_clockify/streams.py

```diff
@@ -0,0 +1,120 @@
+
+from abc import ABC
+from typing import Any, Iterable, Mapping, MutableMapping, Optional
+
+from airbyte_cdk.models import SyncMode
+from airbyte_cdk.sources.streams.http import HttpStream
+
+import requests
+
+
+# Basic full refresh stream
+class ClockifyStream(HttpStream, ABC):
+    url_base = "https://api.clockify.me/api/v1/"
+    page_size = 3
+    page = 1
+    primary_key = None
+
+    def __init__(self, workspaceId: str, **kwargs):
+        super().__init__(**kwargs)
+        self.workspaceId = workspaceId
+
+    def next_page_token(self, response: requests.Response) -> Optional[Mapping[str, Any]]:
+        next_page = response.json()
+        self.page = self.page + 1
+        if next_page:
+            return {"page": self.page}
+
+    def request_params(
+        self, next_page_token: Mapping[str, Any] = None, **kwargs
+    ) -> MutableMapping[str, Any]:
+        params = {
+            "page-size": self.page_size,
+        }
+
+        if next_page_token:
+            params.update(next_page_token)
+
+        return params
+
+    def parse_response(self,
+                       response: requests.Response, **kwargs
+                       ) -> Iterable[Mapping]:
+        yield from response.json()
+
+
+class Users(ClockifyStream):
+    def path(self, **kwargs) -> str:
+        return f"workspaces/{self.workspaceId}/users"

```

Comment:
Please enable cache for this stream, so when reading the `TimeEntrie`won't read a second time.